questionnaires/templates/questionnaires/tags/submit_button.html

```diff
@@ -3,7 +3,7 @@
 <button type="submit" class="cust-btn cust-btn--dark survey-page__btn questionnaire-submit-btn">
     <span>
         {% if fields_step.paginator.num_pages == fields_step.number %}
-            {{ page.submit_button_text }}
+            {% translate page.submit_button_text %}
         {% else %}
             {% translate "Next" %}
         {% endif %}
```

Comment:
I think this a good opportunity to remove this conditional from the template. My preference would be for the submit button label to be passed into the template via context. This would allow the logic to be tested by inspecting the context.